esprima.js

```diff
@@ -1274,27 +1281,13 @@
 
                     default:
                         if (isOctalDigit(ch)) {
```

Comment:
`\08` is disallowed always. Doesn't matter if it's in sloppy mode, in a browser, in a string or regexp or template.